app/kumactl/cmd/get/testdata/get-mesh.golden.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Can we get rid of this separator in every get-X.golden.yaml?

Reply:
was using a formatter will remove it